src/assets/sass/fabric-icons-inline.scss

```diff
@@ -1,139 +1,141 @@
+.ms-Icon--Label:before { @include ms-Icon--Label }
+.ms-Icon--Info:before { @include ms-Icon--Info }
+.ms-Icon--Documentation:before { @include ms-Icon--Documentation }
+.ms-Icon--AddTo:before { @include ms-Icon--AddTo }
+.ms-Icon--Hide3:before { @include ms-Icon--Hide3 }
+.ms-Icon--WarningSolid:before { @include ms-Icon--WarningSolid }
+.ms-Icon--BranchMerge:before { @include ms-Icon--BranchMerge }
+.ms-Icon--StatusCircleCheckmark:before { @include ms-Icon--StatusCircleCheckmark }
+.ms-Icon--AlertSolid:before { @include ms-Icon--AlertSolid }
+.ms-Icon--Plug:before { @include ms-Icon--Plug }
+.ms-Icon--PlugConnected:before { @include ms-Icon--PlugConnected }
+.ms-Icon--ChevronDown:before { @include ms-Icon--ChevronDown }
+.ms-Icon--ChevronUp:before { @include ms-Icon--ChevronUp }
+.ms-Icon--Edit:before { @include ms-Icon--Edit }
+.ms-Icon--Add:before { @include ms-Icon--Add }
+.ms-Icon--Cancel:before { @include ms-Icon--Cancel }
+.ms-Icon--More:before { @include ms-Icon--More }
+.ms-Icon--Link:before { @include ms-Icon--Link }
+.ms-Icon--Settings:before { @include ms-Icon--Settings }
+.ms-Icon--ReceiptProcessing:before { @include ms-Icon--ReceiptProcessing }
+.ms-Icon--Filter:before { @include ms-Icon--Filter }
+.ms-Icon--ZoomOut:before { @include ms-Icon--ZoomOut }
+.ms-Icon--CheckboxComposite:before { @include ms-Icon--CheckboxComposite }
+.ms-Icon--CheckMark:before { @include ms-Icon--CheckMark }
+.ms-Icon--Up:before { @include ms-Icon--Up }
+.ms-Icon--Down:before { @include ms-Icon--Down }
+.ms-Icon--Delete:before { @include ms-Icon--Delete }
+.ms-Icon--Cloud:before { @include ms-Icon--Cloud }
+.ms-Icon--Search:before { @include ms-Icon--Search }
+.ms-Icon--Refresh:before { @include ms-Icon--Refresh }
+.ms-Icon--Share:before { @include ms-Icon--Share }
+.ms-Icon--Download:before { @include ms-Icon--Download }
+.ms-Icon--Help:before { @include ms-Icon--Help }
+.ms-Icon--Rename:before { @include ms-Icon--Rename }
+.ms-Icon--ZoomIn:before { @include ms-Icon--ZoomIn }
+.ms-Icon--View:before { @include ms-Icon--View }
+.ms-Icon--MapLayers:before { @include ms-Icon--MapLayers }
+.ms-Icon--Tag:before { @include ms-Icon--Tag }
+.ms-Icon--Copy:before { @include ms-Icon--Copy }
+.ms-Icon--ChevronLeft:before { @include ms-Icon--ChevronLeft }
+.ms-Icon--ChevronRight:before { @include ms-Icon--ChevronRight }
+.ms-Icon--Home:before { @include ms-Icon--Home }
+.ms-Icon--KeyPhraseExtraction:before { @include ms-Icon--KeyPhraseExtraction }
+.ms-Icon--Insights:before { @include ms-Icon--Insights }
+.ms-Icon--MachineLearning:before { @include ms-Icon--MachineLearning }
+.ms-Icon--TagGroup:before { @include ms-Icon--TagGroup }
+.ms-Icon--BookAnswers:before { @include ms-Icon--BookAnswers }
+.ms-Icon--ChromeRestore:before { @include ms-Icon--ChromeRestore }
+.ms-Icon--ChromeMinimize:before { @include ms-Icon--ChromeMinimize }
+.ms-Icon--System:before { @include ms-Icon--System }
+.ms-Icon--SquareShape:before { @include ms-Icon--SquareShape }
```

Comment:
can we append to this list instead of replacing it every time. I can't tell what's added or deleted

Reply:
@stew-ro  I thought, you told me that we have to replace this file - when we adding icons.